pkg/util/ghactions/runner.go

```diff
@@ -75,3 +86,10 @@ func GetActionRunnerSnippetLinux() string {
 	}
 	return util.IfNillable(args != nil, snippetLinux, "")
 }
+
+func GetActionRunnerSnippetMacos() string {
+	return util.If(args != nil,
```

Comment:
You should use same syntax as others with `util.IfNillable` otherwise f not ghtunner params passed it will panic (I think)

Reply:
ahh, yes